mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/work/SearchIndexingPlanExecutionReport.java

```diff
@@ -21,6 +23,6 @@ public interface SearchIndexingPlanExecutionReport {
 	 * @return A list of references to entities that may not be indexed correctly as a result of the failure.
 	 * Never {@code null}, but may be empty.
 	 */
-	List<?> failingEntities();
+	List<? extends EntityReference> failingEntities();
```

Comment:
This changes the API, but it's a new interface anyway. The only question I wonder is if we should mark it as `@Incubating`?

Reply:
Technically this is a backwards-compatible change :) That's why I insisted on using wildcards here.

True, the `@Incubating`  annotation was missing, I'll add it.